yarn.lock

```diff
@@ -6806,13 +6806,20 @@ __metadata:
   languageName: node
   linkType: hard
 
-"discord-api-types@npm:^0.33.3, discord-api-types@npm:^0.33.5":
+"discord-api-types@npm:^0.33.3":
```

Comment:
What's...still using 0.33.3 in the project? Can you run yarn why on it? 😓

Reply:
![image](https://user-images.githubusercontent.com/74945038/177191935-dc87b629-417e-44da-8751-9e12a008d896.png)
should I nuke yarn.lock and regen it?

Reply:
Why is rest coming from npm...

Reply:
bcz /proxy don't uses yarn workspace for publish